docs/docs/pair-programming.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Open Community Pair Programming Sessions
+---
+
+The best part of open source is the community, and every community is stronger when it works together. To help build the strongest community possible, Gatsby is offering free [one-hour pair programming sessions][cal] to anyone and everyone in the open source community.
+
+## How community pair programming sessions work
+
+1.  Sign up for any available slot on [the open pairing calendar][cal]
+2.  You’ll be paired with one of the Gatsby Inkteam
+3.  Join the video meeting during your time slot
+4.  Build, learn, or discuss your Gatsby or other OSS project!
+
+### What we expect in pair programming sessions
+
+- Open community pair programming sessions are _only_ for Gatsby and/or open source projects
+- All participants are expected to adhere to [Gatsby’s code of conduct](/docs/code-of-conduct/)
+- We will ask if it’s okay to record our session; you are _not_ required to let us record
+- If the session is recorded, we may post the recording on [Gatsby’s YouTube channel](https://www.youtube.com/channel/UCjnp770qk7ujOq8Q9wiC82w)
```

Comment:
I will add here that its nice if you have some problem or something in mind that you want to work on (of course related to Gatsby) but that its not required. what do you think @jlengstorf ?